simphony/cuds/simulation.py

```diff
@@ -0,0 +1,42 @@
+"""Contains simulation controllers."""
+from . import CUDS
+from ..engine import get_wrapper
+
+
+class WrapperFactory(object):
+    """Creates wrappers."""
+    @staticmethod
+    def create(wrapper_name, cuds):
+        """Create a wrapper instance of the given name.
+
+        Parameters
+        ----------
+        cuds: CUDS
+            a cuds object containing simulation data
+        wrapper_name: str
+            A SimPhoNy wrapper name, should be of of registered engines.
+        """
+        wrapper = get_wrapper(wrapper_name)
+        if not wrapper:
+            raise Exception('Unknown wrapper: %s' % wrapper_name)
```

Comment:
It would be helpful to print the list of known `wrapper_name` in the exception message.